### PR TITLE
Added Properties

### LINKED ISSUE #7: add properties
setup the properties for the linked list class that was inherited mentioned in #6 and initialise where appropriated

### PR DESCRIPTION
**Pull request by mofirojean**
fixes #7 
Created a node class
Added some linked list property in the LinkedList class
Changed the zero in the size() method and assigned it the Size property

### CODE REVIEW

**Comment by kimbifavour** [comment]
I can't see the node class. You pushed just the LinkedListImpl class

**Review by kimbifavour — CHANGES_REQUESTED** [review]
Add the node class

**Review by kimbifavour — APPROVED** [review]

**Review by kimbifavour — APPROVED** [review]

File: src/com/kanyelings/LinkedListImpl.java [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 public class LinkedListImpl implements ILinkedList{
+    int size = 0;   // size of the linked list
+    Node head;      // points the head of the linked list
+    Node tail;      // points the tail
+
     @Override
     // access methods
     public int size() { // returns the number of elements in the list
-        return 0;
+        return size;
```
Comment (elroykanye) [inline src/com/kanyelings/LinkedListImpl.java:11]: size is not a private property. I don't see the need for a a method that returns it again. That's redundancy.

Also, size could be easily accessed and modified in an object causing it not to reflect the true nature of the list.

**Review by elroykanye — CHANGES_REQUESTED** [review]
Looks good, but not very.
You may have to check out the meanings of various access modifiers and their example use cases so you know when and how to use what. 😊

File: src/com/kanyelings/LinkedListImpl.java (hunk 1 above)
Comment (mofirojean) [reply src/com/kanyelings/LinkedListImpl.java:11]: Kanye i don't quite understand, please can you explain

Comment (elroykanye) [reply src/com/kanyelings/LinkedListImpl.java:11]: Do some research on getters and setters.

Remember, this is java not python.

**Comment by elroykanye** [comment]
Are you still working on this?

File: src/com/kanyelings/LinkedListImpl.java (hunk 1 above)
Comment (mofirojean) [reply src/com/kanyelings/LinkedListImpl.java:11]: Ok

Comment (elroykanye) [reply src/com/kanyelings/LinkedListImpl.java:11]: `very secure` :)

**Review by elroykanye — APPROVED** [review]

File: src/com/kanyelings/LinkedListImpl.java (hunk 1 above)
Comment (mofirojean) [reply src/com/kanyelings/LinkedListImpl.java:11]: can i delete the branch now

**Comment by mofirojean** [comment]
Kanye can i now delete the branch?

File: src/com/kanyelings/LinkedListImpl.java (hunk 1 above)
Comment (elroykanye) [reply src/com/kanyelings/LinkedListImpl.java:11]: Depends on you 🙂

**Pull request merged**